akka-docs/src/main/paradox/stream/operators/StreamConverters/asInputStream.md

```diff
@@ -25,3 +25,10 @@ The `InputStream` will be ended when the stream flowing into this `Sink` complet
 **backpressures** when no read is pending on the `InputStream`
 @@@
 
+## Example
+
```

Comment:
Please include a short textual explanation of what the example does here

Reply:
Done